decidim-core/app/views/layouts/decidim/_application.html.erb

```diff
@@ -3,7 +3,11 @@
   title: current_organization.name,
   url: request.original_url,
   twitter_handler: current_organization.twitter_handler,
-  image_url: current_organization.homepage_image.url
+  image_url: Decidim::ContentBlock.published.find_by(
```

Comment:
Maybe we should add this logic to the organization, so the external api doesn't change.